src/parse/errors.rs

```diff
@@ -411,68 +411,17 @@ impl Error {
         process::exit(0);
     }
 
-    #[doc(hidden)]
+    /// Render and write the error into `w`.
     pub fn write_to<W: Write>(&self, w: &mut W) -> io::Result<()> {
         write!(w, "{}", self.message)
     }
 
-    #[doc(hidden)]
-    pub fn group_conflict<O, U>(
```

Comment:
I also found during my color refactor that we don't use this. But can you please leave this function for now? I will address it in my branch.

Reply:
Only this one or also `argument_conflict`?

Reply:
You haven't changed anything (other than `crate`) for `argument_conflict`. That is okay. Just don't remove this one function and add `crate` here.

Reply:
Oops, I was talking about `io_error`. Do you need it too or I can leave it removed?

Reply:
Keep it removed please.